src/main/webapp/components/filters/property-filter.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+angular.module('managementConsole.filters', [])
+    .filter('customPropertyFilter', function (){
```

Comment:
File is named property-filter, so I would probably name the filter the same way: word "custom" is not really descriptive anyway, it does not bring much value I think.